README.md

```diff
@@ -94,7 +94,7 @@ $ make test
 The majority of tests in the provider are Acceptance Tests - which provisions real resources in Azure. It's possible to run the entire acceptance test suite by running `make testacc` - however it's likely you'll want to run a subset, which you can do using a prefix, by running:
 
 ```
-make testacc TESTARGS='-run=TestAccAzureRMResourceGroup'
+make testacc TESTARGS=-run=TestAccAzureRMResourceGroup
```

Comment:
can we revert this? whilst the quotes don't matter on macOS and Linux - they're mostly here for Windows support (unless you're using Windows and this is working for you, in which case ignore this 🙃)

Reply:
I was giving me issues in Mac, that is why I thought it was a typo. Fixed in a new push.